test/framework/cluster_helpers.go

```diff
@@ -144,9 +143,6 @@ type DeleteClusterInput struct {
 
 // DeleteCluster deletes the cluster and waits for everything the cluster owned to actually be gone.
 func DeleteCluster(ctx context.Context, input DeleteClusterInput) {
-	if options.SkipResourceCleanup {
-		return
-	}
```

Comment:
Are these ok to remove?

Reply:
Yes, the options.SkipResourceCleanup is deprecated and it is being removed in this PR (it was replaced by a flag in the e2e suite)